### DISCUSSION TITLE
Can we skip some part of the video while playing it. If yes then how?

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Ed/Community#467] [General Discussion]
Actually, I was trying to skip some parts of the video while playing it but whenever I try to skip, it starts from the beginning.
I have uploaded videos through the **upload large video and upload content.**
Made it Live and while consuming I was trying to skip some parts of it but could not.
**CAN WE SKIP IT? IF YES THEN PLEASE HELP US DO THAT.**
![image](https://user-images.githubusercontent.com/109504832/233016487-cee33c03-4492-42fc-a042-10ec8d91342e.png)
**This is a video link**
https://user-images.githubusercontent.com/109504832/233018700-ca011fac-f0ac-43c6-8127-b69ee3661e90.mp4
[@vinukumar-vs](https://github.com/vinukumar-vs) [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)

**Comment by ashokreddy1208** [comment]
@sachinb80 We've tried this scenario in the SunbirdED 5.2  and 5.1 releases and it is working fine. Here is a recording for your reference. btw., which version of ED are you using?

[Sunbird ED Video (1).webm](https://user-images.githubusercontent.com/58033030/233286703-91451f5d-b219-41a8-bfe4-53553556f0c7.webm)

[@vinukumar-vs](https://github.com/vinukumar-vs) [@rajeevsathish](https://github.com/rajeevsathish) @[rajeshkumaravel](https://github.com/rajeshkumaravel) [@Krishnaj20](https://github.com/Krishnaj20)

**Reply by sachinb80** [reply]
@ashokreddy1208 We are using the Sunbird 4.6 version.

**Reply by Krishnaj20** [reply]
@sachinb80 - as informed by Ashok, it is working fine in the supported versions of ED. Community can provide further help if you can share more detailed analysis such as debug logs etc.

Please take help of @vinod303 for getting these details.

**Reply by sachinb80** [reply]
@Krishnaj20 sure
@vinod303 Can you please help in debugging this?
